test/e2e/framework/test_context.go

```diff
@@ -26,12 +26,13 @@ import (
 )
 
 type TestContextType struct {
-	KubeConfig            string
-	KubeContext           string
-	KubeAPIContentType    string
-	KubeVolumeDir         string
-	CertDir               string
-	Host                  string
+	KubeConfig         string
+	KubeContext        string
+	KubeAPIContentType string
+	KubeVolumeDir      string
+	CertDir            string
+	Host               string
+	// TODO: Deprecating this over time... instead just use gobindata_util.go.
```

Comment:
Specifically `RepoRoot` is what we're trying to deprecate, right?  Is there an issue we should tie to?